integr8ly/integreatly-operator
# add multitenant-managed-api installation type

# Description
Add new install type "multitenant-managed-api"

# Verification
On BYOC cluster run:
- run `INSTALLATION_TYPE=multitenant-managed-api make cluster/prepare/local` 
- run `INSTALLATION_TYPE=multitenant-managed-api make code/run`
- set the cluster storage to false
- Confirm that RHMI CR installation type is multitenant-managed-api and installation completes successfully. 
- Navigate to prometheus and confirm that only deadmans snitch alert is firing
- Navigate to Grafana and confirm that dashboards are available
- Make sure that 3scale and keycloak are available
- Uninstall by deleting RHMI CR
- Confirm that RHOAM + RHMI e2e tests are successful on this PR
